test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12639,7 +12639,26 @@ def isUp(self):
         """
         Check for PBS mom up
         """
-        return super(MoM, self)._isUp(self)
+        # Poll for few seconds so we don't miss any signals sent to
+        # mom
+        for _ in range(5):
+            rv = super(MoM, self)._isUp(self)
+            if rv:
+                try:
+                    nodes = self.server.status(NODE, id=self.shortname)
+                    if nodes:
+                        n_stat = self.server.expect(NODE,
```

Comment:
No need to take the value of expect().  You don't use it.

Reply:
Yes. Removed it